gems/sorbet-runtime/test/types/struct.rb

```diff
@@ -9,14 +9,4 @@ class Opus::Types::Test::StructValidationTest < Critic::Unit::UnitTest
     end
     assert_includes(err.message, "is a subclass of T::Struct and cannot be subclassed")
   end
-
-  it "forbids declaring a Struct as abstract" do
-    err = assert_raises(RuntimeError) do
-      Class.new(T::Struct) do
-        extend T::Helpers
-        abstract!
-      end
-    end
-    assert_includes(err.message, "is a subclass of T::Struct and cannot be declared abstract")
-  end
```

Comment:
@azdavis same for this one

Reply:
If we take out the check, then we should take out the test testing to see if the check is there, so yes.